src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -199,25 +231,37 @@ export class ProgressIndicator extends Component {
         };
   }
 
-  renderSteps = () =>
-    React.Children.map(this.props.children, (child, index) => {
+  renderSteps = () => {
+    const { onChange } = this.props;
+
+    return React.Children.map(this.props.children, (child, index) => {
+      // only setup click handlers if onChange event is passed
+      const onClick =
+        onChange && index > -1 ? () => onChange(index) : undefined;
```

Comment:
Do you anticipate `index` being negative in `React.Children.map()` callback?

Reply:
you're right, great catch I've removed that in the latest commit.